framework/vulkan_sample.cpp

```diff
@@ -178,7 +183,13 @@ bool VulkanSample::prepare(Platform &platform)
 		debug_utils = std::make_unique<DummyDebugUtils>();
 	}
 
-	device = std::make_unique<vkb::Device>(gpu, surface, std::move(debug_utils), get_device_extensions());
+	// @todo: add callback for sample
```

Comment:
similar here, happy with the override just need to clean-up the todo and maybe move the default call to `create_device`